nntrainer/layers/time_dist.h

```diff
@@ -93,6 +93,24 @@ class TimeDistLayer : public Layer {
    */
   std::shared_ptr<Layer> getDistLayer() { return dist_layer; };
 
+  /**
+   * @brief     get transposed Tensor
+   * @param[in] m Tensor
+   * @retval Tensor transposed Tensor
+   */
+  Tensor transposeTensor(Tensor &m);
+
+  /**
+   * @brief  calculate the pointer of each input and output tensors
+   */
+  void setPosition();
+
+  /**
+   * @brief  Transpose Input and Output Tensors to avoid duplicatation becuase
```

Comment:
Could you add a comment how transpose is done here as well?

Reply:
done.